src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/IncrementingPollingCounter.cs

```diff
@@ -69,6 +70,9 @@ internal void UpdateMetric()
             }
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
+            Justification = "IncrementingCounterPayload type has been trimmer safe")]
```

Comment:
Same message about DynamicDependency. I'll stop commenting on this in other places, and assume you will fix it everywhere.